modules/importing-repositories/procedure-enabling-the-bulk-import-from-github-feature.adoc

```diff
@@ -0,0 +1,37 @@
+[id="enabling-ang-giving-access-to-the-bulk-import-feature"]
+= Enabling and giving access to the Bulk Import feature
+
+.Procedure
+. To give {product-short} access to GitHub, link:{linkgettingstartedguide}#configuring-github-integration[configure GitHub integration].
+
+. The Bulk Import plugins are installed but disabled by default.
+To enable the `./dynamic-plugins/dist/janus-idp-backstage-plugin-bulk-import-backend-dynamic` and `./dynamic-plugins/dist/janus-idp-backstage-plugin-bulk-import` plugins,
+edit your `dynamic-plugins.yaml` with following content.
+See link:{plugins-configure-book-url}[{plugins-configure-book-title}].
++
+.`dynamic-plugins.yaml` fragment
+[source,yaml]
+----
+global:
```

Comment:
IMO, this example is too-specific to the Helm Chart (specifically the `global.dynamic` field). 
I'll suggest removing the `global.dynamic` container from this YAML.
But actually, I just came across this nice doc on how to enable a dynamic plugin using either Helm or the Operator: https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.3/html/installing_and_viewing_dynamic_plugins/index
I think we could link to this doc instead. WDYT?